### PR TITLE
oci-cli 2.12.4 (new formula)

### PR DESCRIPTION
**Pull request by kernleee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
You'll need to wait for https://github.com/Homebrew/brew/pull/7953 to be merged.

**Comment by kernleee** [comment]
Are dual licenses not supported?

**Comment by SMillerDev** [comment]
Not until that PR is merged.

File: Formula/oci-cli.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class OciCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Oracle Cloud Infrastructure CLI"
+  homepage "https://docs.cloud.oracle.com/iaas/Content/API/Concepts/cliconcepts.htm"
+  url "https://github.com/oracle/oci-cli/archive/v2.12.3.tar.gz"
+  sha256 "c0879daee931cd03cf0ea06661a79f86c13b803b778f32a1e7938653e100c566"
+  license "UPL-1.0"
+  license "Apache-2.0"
```
Comment (lionellloh) [inline Formula/oci-cli.rb:9]:
```suggestion
```

**Comment by kernleee** [comment]
A question for the Homebrew team. Would it be possible to choose one license for now while the dual license PR is in progress and then update this formula later with the second license?

**Comment by SMillerDev** [comment]
Seems okay to me, can you leave the real licenses in a comment below it?

**Comment by kernleee** [comment]
Sure, do you mean like

license "UPL-1.0"
# license ["UPL-1.0", "Apache-2.0"]

**Comment by SMillerDev** [comment]
Seems good, can you put the brew PR url behind it?

**Comment by kernleee** [comment]
Had to change the comment slightly as the license still got caught in the tests. Let me know if thats fine.

**Review by chenrui333 — APPROVED** [review]
👍  lgtm

**Comment by kernleee** [comment]
Can this possibly be merged?

**Comment by chenrui333** [comment]
> Can this possibly be merged?

thanks for the reminder, merging.

**Comment by chenrui333** [comment]
@kernleee, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**